README.md

```diff
@@ -1,94 +1,125 @@
+  - macro_namespace: dbt_utils
+    search_order: ['spark_utils', 'dbt_utils']
+```
 
+## Step 2: Install the package
+Include the following snapchat_ads_source package version in your `packages.yml` file:
+> TIP: Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
 ```yaml
 packages:
   - package: fivetran/snapchat_ads
-    version: [">=0.3.0", "<0.4.0"]
-```
+    version: [">=0.4.0", "<0.5.0"]
 
-## Configuration
+```
+## Step 3: Configure your variables
 
-By default, this package will look for your Snapchat Ads data in the `snapchat_ads` schema of your [target database](https://docs.getdbt.com/docs/running-a-dbt-project/using-the-command-line-interface/configure-your-profile). If this is not where your Snapchat Ads data is, please add the following configuration to your 
-`dbt_project.yml` file:
+### Define database and schema variables
+By default, this package runs using your destination and the `snapchat_ads` schema. If this is not where your Snapchat Ads data is (for example, if your Snapchat Ads schema is named `snapchat_ads_fivetran`), add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
-config-version: 2
-
 vars:
-    snapchat_schema: your_schema_name
-    snapchat_database: your_database_name 
+    snapchat_ads_database: your_destination_name
+    snapchat_ads_schema: your_schema_name 
 ```
 
-For additional configurations for the source models, visit the [Snapchat Ads source package](https://github.com/fivetran/dbt_snapchat_ads_source).
-
-### Changing the Build Schema
-
-By default, this package will build the Snapchat Ads staging models within a schema titled (`<target_schema>` + `_stg_snapchat_ads`) and the final Snapchat Ads models within a schema titled (`<target_schema>` + `_snapchat_ads`) in your target database. If this is not where you would like your modeled Snapchat data to be written to, add the following configuration to your `dbt_project.yml` file:
+## (Optional) Step 4: Additional configurations
+### Passing Through Additional Metrics
+By default, this package will select fields from the source tables to pass into the staging models, that are shown in the existing staging models. If you would like to pass through additional metrics to the staging models, add the following configuration to your `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
+vars:
+  snapchat_ads__ad_hourly_passthrough_metrics: ['the', 'list', 'of', 'metric', 'columns', 'to', 'include'] # from snapchat_ads.ad_hourly_report
+  snapchat_ads__ad_squad_hourly_passthrough_metrics: ['the', 'list', 'of', 'metric', 'columns', 'to', 'include'] # from snapchat_ads.ad_squad_hourly_report
+  snapchat_ads__campaign_hourly_report_passthrough_metrics: ['the', 'list', 'of', 'metric', 'columns', 'to', 'include'] # from snapchat_ads.campaign_hourly_report
+```
+    
```

Comment:
Can we also add in the [change the source table references](https://github.com/fivetran/dbt_snapchat_ads_source/tree/feature/v2_updates#change-the-source-table-references) section to this README.